build-tools/packages/build-infrastructure/src/test/packageJsonUtils.test.ts

```diff
@@ -20,6 +20,7 @@ import { testDataPath } from "./init.js";
 /**
  * A transformer function that does nothing.
  */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Curious why do this here instead of the underscore prefix.

Reply:
I can't figure out why, but the lint rule gets tripped even when disabled in the config. I ended up just removing the placeholder args completely.